texar/torch/modules/embedders/embedders.py

```diff
@@ -258,6 +258,9 @@ def num_embeddings(self) -> int:
 
     @property
     def output_size(self) -> int:
+        r"""The feature size of :meth:`forward` output :attr:`outputs`.
+        Same as embedding dimension's last element or itself.
```

Comment:
I don't understand this sentence. Could you rephrase it? And same for the position embedders.

Reply:
We might not need to mention `outputs` here because the `forward` method returns only one value.